common/djangoapps/util/organizations_helpers.py

```diff
@@ -103,3 +103,17 @@ def organizations_enabled():
     Returns boolean indication if organizations app is enabled on not.
     """
     return settings.FEATURES.get('ORGANIZATIONS_APP', False)
+
+
+def get_course_secondary_organizations_names(course_id, primary_org_name):
```

Comment:
Just a Suggestion: the function name makes a bit ambiguity, shouldn't it be something like `get_secondary_org_names_for_course`?